.github/workflows/sycl-post-commit.yml

```diff
@@ -53,14 +68,21 @@ jobs:
             runner: '["Linux", "gen12"]'
             extra_lit_opts: --param gpu-intel-gen12=True
             target_devices: level_zero:gpu;opencl:fpga
+            reset_gpu: true
```

Comment:
Is `reset_gpu: true` in Gen12 and Arc jobs really needed to fix HIP postcommit?

Reply:
no, it was already getting set to true always in the below `with` part.
For some things you can only set the variable inside the matrix and do a `||` inside the with part, but since `reset_gpu` is a boolean, that trick doesn't work, so I needed to define it in every matrix entry.

The value of `reset_gpu` was and remains `true` for the non-HIP E2E tests.